filer/private/sass/components/_drag-and-drop.scss

```diff
@@ -334,3 +334,6 @@ form .form-row {
     text-overflow: ellipsis;
     white-space: nowrap;
 }
+&.reset-hover {
```

Comment:
why the `&`? I don't think there is any nesting involved, no parent above in the file